libcudacxx/include/cuda/__bit/bitfield.h

```diff
@@ -21,60 +21,96 @@
 #  pragma system_header
 #endif // no system header
 
-#include <cuda/__ptx/instructions/bmsk.h>
-#include <cuda/std/__concepts/concept_macros.h>
+#include <cuda/__bit/bitmask.h>
+#include <cuda/std/__limits/numeric_limits.h>
+#include <cuda/std/__type_traits/conditional.h>
 #include <cuda/std/__type_traits/is_constant_evaluated.h>
 #include <cuda/std/__type_traits/is_unsigned_integer.h>
+#include <cuda/std/cstdint>
 #include <cuda/std/limits>
 
 _LIBCUDACXX_BEGIN_NAMESPACE_CUDA
 
+#if defined(__CUDA_ARCH__)
```

Comment:
We should never use open coded `__CUDA_ARCH__` please replace with e.g `__cccl_ptx_isa` or `_CCCL_HAS_CUDA_COMPILER`